back/app/services/permissions_service.rb

```diff
@@ -63,7 +63,7 @@ def requirements(permission, user)
 
   def requirements_fields(permission)
     if permission.global_custom_fields
-      CustomField.registration
+      CustomField.registration.enabled
```

Comment:
@luucvanderzee I've included a fix for the requirements API returning disabled fields in this PR. Should hopefully be merged on Monday